frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -12,36 +12,53 @@ import {
    VStack,
 } from '@chakra-ui/react';
 import { formatFacetName } from '@helpers/algolia-helpers';
-import { WikiInfoEntry } from '@models/product-list/types';
+import { ProductList } from '@models/product-list';
 import * as React from 'react';
 import { useHits, useRefinementList } from 'react-instantsearch-hooks-web';
-import { RefinementList } from './RefinementList';
+import { FacetFilter } from './FacetFilter';
 import { useCountRefinements } from './useCountRefinements';
 import { useFilteredFacets } from './useFacets';
 
 type FacetsAccordianProps = {
-   wikiInfo: WikiInfoEntry[];
+   productList: ProductList;
 };
 
-export function FacetsAccordion(props: FacetsAccordianProps) {
-   const { wikiInfo } = props;
-   const facets = useFilteredFacets(wikiInfo);
+const initialExpandedFacets = ['facet_tags.Item Type'];
+
+export function FacetsAccordion({ productList }: FacetsAccordianProps) {
+   const facets = useFilteredFacets(productList);
    const countRefinements = useCountRefinements();
+   const [indexes, setIndexes] = React.useState<number[]>(() => {
+      return initialExpandedFacets
+         .map((expandedFacet) =>
+            facets.findIndex((facet) => facet === expandedFacet)
+         )
+         .filter((index) => index >= 0);
+   });
+
+   const handleChangeIndexes = React.useCallback((indexes: number[]) => {
+      setIndexes(indexes);
+   }, []);
 
    return (
       <Accordion
          allowMultiple
+         index={indexes}
+         onChange={handleChangeIndexes}
```

Comment:
Is the argument to the `onChange` callback always a `number[]`, or could it be a `number` too? I see both when I look at Chakra's `<Accordian>` source:

https://github.com/chakra-ui/chakra-ui/blob/e3e02a4f4c81848991b301b07080b9a97e6b4964/packages/accordion/src/use-accordion.ts#L40

Reply:
Depends on how `Accordion` is used. When used with `allowMultiple` prop like we're doing, then the argument it's always an array